packages/@webex/plugin-cc/src/services/task/types.ts

```
@@ -1,512 +1,870 @@
     broadcast: boolean;
+    /** Type of cleanup being performed */
     type: string;
   };
 };
 
 /**
- * Response type for the task public methods
+ * Response type for task public methods
+ * Can be an AgentContact object, Error, or void
  */
 export type TaskResponse = AgentContact | Error | void;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)